samples/client/petstore/java/okhttp-gson-dynamicOperations/src/main/java/org/openapitools/client/api/PetApi.java

```diff
@@ -272,6 +272,11 @@ public okhttp3.Call deletePetCall(Long petId, String apiKey, final ApiCallback _
             localVarHeaderParams.put("Content-Type", localVarContentType);
         }
 
+        if (apiKey != null) {
+            localVarHeaderParams.put("api_key", localVarApiClient.parameterToString(apiKey));
+        }
```

Comment:
I think this change was not intended and should be prevented, right? I.e. when `dynamicOperations` is set to `true`, these headers should not be added.

Reply:
This was unintended, I'll remove them if `dynamicOperations` is true